client/my-sites/stats/post-performance/index.jsx

```diff
@@ -112,12 +116,31 @@ const StatsPostPerformance = React.createClass( {
 										} )
 									}
 								</p>
-							) : null
-						}
-					</div>
-					<StatsTabs>
-						{ this.buildTabs( summaryUrl ) }
-					</StatsTabs>
+							</div>
+						) : null
+					}
+					{ ! loading && ! post
+						? (
+							<div className="module-content-text is-empty-message is-error">
+								<p className="stats-post-performance__no-posts-message">
+									{ this.translate( 'You haven\'t published any posts yet.' ) }
+								</p>
+								<div className="stats-post-performance__start-post">
+									<a href={ newPostUrl } className="button is-primary">
```

Comment:
I think @jancavan had brought this up in a prior comment, but perhaps we can just utilize the [button component](https://github.com/Automattic/wp-calypso/tree/master/client/components/button) here.